Sources/SuperDelegate.swift

```diff
@@ -55,7 +55,6 @@ open class SuperDelegate: NSObject, UIApplicationDelegate {
             return
         }
         
-        mainWindow.frame = UIScreen.main.bounds
```

Comment:
I just tested this line on an iOS 8 simulator (plus iPad and a few others sims). I'm shocked this works. But, it does! So let's back out the platform bump.